src/site/layouts/page.drupal.liquid

```diff
@@ -13,24 +9,28 @@
         </div>
       {% endif %}
 
-      <article class="usa-content">
-        <h1>{{ title }}</h1>
-        <div class="usa-font-lead">
-          {{ fieldIntroText }}
-        </div>
-        {% for block in fieldContentBlock %}
-          {% assign bundleComponent = "src/site/paragraphs/" | append: block.entity.entityBundle %}
-          {% assign bundleComponentWithExtension = bundleComponent | append: ".drupal.liquid" %}
-          {% include {{ bundleComponentWithExtension }} with block = block %}
-        {% endfor %}
-      </article>
+      {% include 'src/site/layouts/sidebar_navigation.drupal.liquid' %}
```

Comment:
I think the only change in this file should be that this include replaces the `<nav class="usa-width-one-fourth va-sidebarnav" id="va-detailpage-sidebar"></nav>` line.